conmon/conmon.c

```diff
@@ -1351,6 +1375,11 @@ int main(int argc, char *argv[])
 		pexit("Failed to fork the create command");
 	} else if (!create_pid) {
 		/* FIXME: This results in us not outputting runc error messages to crio's log. */
```

Comment:
You probably want to keep this comment close to the standard-stream handling below.